docs/src/getting-started.md

```diff
@@ -0,0 +1,178 @@
+# Getting Started
+
+## For Users
+
+### Installation
+
+Installing ClimaLand in Julia is very easy. First, download and install Julia
```

Comment:
Describing something as "very easy" is typically not the best for pedagogy. It may be very easy for you, but it is certainly not very easy for my grandma. It might also lead to frustration: how would you feel if you couldn't accomplish a task described as "very easy"? I suggest we avoid language like this when possible to make our docs more welcoming.

Reply:
I agree! I adjusted the text